scripts/release-android-apk.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+
+set -eux
+
```

Comment:
`flutter pub get && flutter pub run build_runner build --delete-conflicting-outputs`

please add it before build.

Reply:
This step is after [releasing to store](https://github.com/linagora/twake-on-matrix/blob/main/.github/workflows/release.yaml#L78-L80), [which already has the dependencies step](https://github.com/linagora/twake-on-matrix/blob/main/scripts/release-playstore-beta.sh#L2). Using it again here will make the job take more time.